features/utils/credentials.ts

```diff
@@ -129,57 +164,85 @@ const generateErrorMessages = (errors: ErrorObject[]): string[] => {
   });
 };
 
-const loadFromKintone: () => Promise<Credential[]> = async () => {
+const loadFromKintone: () => Promise<Credentials> = async () => {
   const kintoneBaseUrl = process.env.TEST_KINTONE_BASE_URL;
-  const kintoneAppId = process.env.TEST_KINTONE_CREDENTIAL_MANAGEMENT_APP_ID;
-  const kintoneApiToken =
+  const kintoneCMAAppId = process.env.TEST_KINTONE_CREDENTIAL_MANAGEMENT_APP_ID;
+  const kintoneCMAApiToken =
```

Comment:
Now CMA (Credential Management App) should be ACMA (App Credential Management App), as same as UCMA.
If the name ACMA and UCMA is not easy-to-read, I propose to use AppCMA and UserCMA.

Reply:
Yes, I think AppCMA and UserCMA are more readable.